IoTCIntegration/package.json

```diff
@@ -8,7 +8,7 @@
     "azure-iot-device": "^1.7.4",
     "azure-iot-device-http": "^1.7.4",
     "request": "^2.88.0",
```

Comment:
Please remove request as well. It's the source of the deprecation.

Reply:
Done